docs/migration_guides/qi_migration.rst

```diff
@@ -0,0 +1,699 @@
+
+        simulator = AerSimulator()
+        qi = QuantumInstance(backend=simulator, shots=200, backend_options={"method": "statevector"})
+        result = qi.execute(circuit).results[0]
+        data = result.data
+        counts = data.counts
+
+        print("Counts: ", counts)
+        print("Data: ", data)
+        print("Result: ", result)
+
+    .. testoutput::
+       :options: +SKIP
+
+        Counts: {'0x3': 200}
+        Data: ExperimentResultData(counts={'0x3': 200})
+        Result: ExperimentResult(shots=200, success=True, meas_level=2, data=ExperimentResultData(counts={'0x3': 200}), header=QobjExperimentHeader(clbit_labels=[['meas', 0], ['meas', 1]], creg_sizes=[['meas', 2]], global_phase=0.0, memory_slots=2, metadata={}, n_qubits=2, name='circuit-112', qreg_sizes=[['q', 2]], qubit_labels=[['q', 0], ['q', 1]]), status=DONE, seed_simulator=3116700546, metadata={'parallel_state_update': 16, 'parallel_shots': 1, 'sample_measure_time': 6.0573e-05, 'noise': 'ideal', 'batched_shots_optimization': False, 'remapped_qubits': False, 'device': 'CPU', 'active_input_qubits': [0, 1], 'measure_sampling': True, 'num_clbits': 2, 'input_qubit_map': [[1, 1], [0, 0]], 'num_qubits': 2, 'method': 'statevector', 'fusion': {'applied': False, 'max_fused_qubits': 5, 'threshold': 14, 'enabled': True}}, time_taken=0.000426016)
+
+
+    **Using Primitives**
+
+    The primitives offer two alternatives for local statevector simulation, one with the Reference primitives
+    and one with the Aer primitives. As mentioned above the closest alternative to ``QuantumInstance.execute()``
+    for sampling is the ``Sampler`` primitive.
+
+    **a. Using the Reference Primitives**
+
+    Basic statevector simulation based on the :class:`qiskit.quantum_info.Statevector` class. Please note that
+    the resulting quasi-probability distribution does not use bitstrings but **integers** to identify the states.
+
+    .. testcode::
+
+        from qiskit import QuantumCircuit
+        from qiskit.primitives import Sampler
+
+        circuit = QuantumCircuit(2)
+        circuit.x(0)
+        circuit.x(1)
+        circuit.measure_all()
+
+        sampler = Sampler(options = {"shots":200})
+        result = sampler.run(circuit).result()
+        quasi_dists = result.quasi_dists
+
+        print("Quasi-dists: ", quasi_dists)
+        print("Result: ", result)
+
+    .. testoutput::
+       :options: +SKIP
+
+        Quasi-dists: [{3: 1.0}]
+        Result: SamplerResult(quasi_dists=[{3: 1.0}], metadata=[{'shots': 200}])
+
+    **b. Using the Aer Primitives**
+
+    Aer simulation following the statevector method. This would be the direct 1-1 replacement of the ``QuantumInstance``
+    example, as they are both accessing the same simulator. For this reason, the output metadata is
+    closer to the Quantum Instance's output. Please note that
+    the resulting quasi-probability distribution does not use bitstrings but **integers** to identify the states.
```

Comment:
You could note the QuasiProb does have methods `binary_probabilities` and `hex_probabilities` if the integer keys are not to your liking.

Reply:
Added note and example